### ISSUE TITLE
Redirect to login when there's no user logged in.

### ISSUE THREAD

**Issue by huevosabio** [huevosabio/windopt#13]
First, we need to adapt satellizer to actually work well with Flask.

Second, we need to not show anything when the user is not logged in and just route to login page. 

Third, when the user logs out it should redirect to the sign in page.

**Comment by huevosabio** [comment]
Ok, so the current issue, last I checked was that satellizer and flask-httpbasicauth are not using the same protocol for authentication. Should we find no way to make them agree, we should favor dropping Satellizer.

**Comment by huevosabio** [comment]
First, let's build the features such that item 2 works (redirect).

**Comment by huevosabio** [comment]
Ok, I've implemented the number 2.

**Comment by huevosabio** [comment]
Upon reviewing the requests sent by my browser, it seems that satellizer is actually not adding the "Authorization" header that is required by Flask... this is odd since Satellizer is supposed to be injecting the header. 

from satellizer:
```javascript
.factory('SatellizerInterceptor', [
      '$q',
      'SatellizerConfig',
      'SatellizerStorage',
      'SatellizerShared',
      function($q, config, storage, shared) {
        return {
          request: function(request) {
            if (request.skipAuthorization) {
              console.log('skipAuthorization')
              return request;
            }

            if (shared.isAuthenticated() && config.httpInterceptor) {
              var tokenName = config.tokenPrefix ? config.tokenPrefix + '_' + config.tokenName : config.tokenName;
              var token = storage.get(tokenName);

              if (config.authHeader && config.authToken) {
                token = config.authToken + ' ' + token;
              }

              request.headers[config.authHeader] = token;
            }

            return request;
          },
          responseError: function(response) {
            return $q.reject(response);
          }
        };
      }])
```

**Comment by huevosabio** [comment]
I believe I found the issue. Satellizer adds the Authorization header ONLY after it has logged in. Flask_HTTPAuth requires the Authorization header... I'm just going to go for the easy way out which is use the code in the Satellizer example (https://github.com/sahat/satellizer/blob/master/examples/server/python/app.py).

**Comment by huevosabio** [comment]
All three requirements are met, the last thing is that on login it should go to the main page.

**Comment by huevosabio** [comment]
The three have been implemented, there remains an issue of redirecting on login, I'll open a new issue on that matter.
